src/SIM/SIMmodal.C

```diff
@@ -67,6 +73,22 @@ SIMmodal::~SIMmodal ()
 }
 
 
+bool SIMmodal::parseParams (const tinyxml2::XMLElement* elem)
+{
+  if (parsed)
+    IFEM::cout <<"\t(skipped)"<< std::endl;
+  else if (!strcasecmp(elem->Value(),"newmarksolver"))
```

Comment:
may as well check for non-null elem or some sca might get nervous.

Reply:
ok, done that too. But there are any other places around where we don't do this check.

Reply:
yeah i'm aware. sometimes i've had to add checks to quell some warnings so just a defensive knee-jerk.